sui/Cargo.toml

```diff
@@ -33,6 +33,8 @@ serde-name = "0.2.0"
 dirs = "4.0.0"
 clap = { version = "3.1.14", features = ["derive"] }
 telemetry_subscribers = { git = "https://github.com/MystenLabs/mysten-infra", rev = "d2976a45420147ad821baae96e6fe4b12215f743" }
+multiaddr = "0.14.0"
+mysten-network = { git = "https://github.com/bmwill/mysten-infra.git", rev = "344471e3d3981585a2be7a5e25ea5fa8ba124a98" }
```

Comment:
Presumably, you could update telemetry-subscribers when you re-jig this with your merged mysten-infra revision.